tests/ci/run_build.py

```diff
@@ -172,6 +172,10 @@ def build_srpm_and_rpm(command, family=None):
     if family is not None:
         rebuild_command[-1] += " --define 'mpi_family %s'" % family
 
+    # Disable parallel builds for boost on aarch64 to avoid OOM
```

Comment:
> At this point we could try with something like if uname -m equals aarch64 and spec file is boost.spec we could add --define _smp_mflags 1. Would be worth a try. Probably in the Python script that handles the rebuilds.

@adrianreber  Is that so? Although I didn't quite understand the meaning of _smp_mflags, I tried it.

But looks like build failure

Reply:
You need to set it to `-j1` and not just `1`.

Reply:
Then I understand! lol!